setupReactBatching.js

```diff
@@ -0,0 +1,4 @@
+const {setBatch} = require('./src/util/Recoil_batch');
+const {unstable_batchedUpdates} = require('react-dom');
+
+setBatch(unstable_batchedUpdates);
```

Comment:
Is this only used for testing?  Could you add some comments how this interacts and works with the call to `setBatch` in `Recoil_index.js` to the `util/ReactBatchedUpdates` version?  Hmm, seems that this is added as a `setupFiles` in the `package.json`.  @mondaychen could you review the packaging part of this?  (Internally we'd have to move this file to scripts)

Reply:
Yeah, we needed `setupReactBatching.js` for testing because the default batching function was being set in `Recoil_index.js` and we were importing individual module files directly in tests, meaning the batching fn wasn't set. But now that that's happening in `Recoil_batching.js`, we don't need this file anymore. 🙌